csrc/executor_kernel_arg.h

```diff
@@ -59,16 +60,19 @@ inline std::string argTypeToString(ArgType type) {
 }
 
 // This should match the tensor used in the code generation (almost exactly)
-template <int ndims, typename nvfuser_index_t>
+template <int ndims, int nalloc, typename nvfuser_index_t>
 struct TensorArgCodegen {
   using index_type = nvfuser_index_t;
 
   void* data;
   std::array<nvfuser_index_t, ndims> size;
-  std::array<nvfuser_index_t, ndims> stride;
+  std::array<nvfuser_index_t, nalloc> stride;
   constexpr int nDims() const {
     return ndims;
   }
+  constexpr int nAllocationDims() const {
```

Comment:
I guess it doesn't matter as it's `constexpr`, but this should be `static`

Reply:
fixed